lib/eco/test/test_eco.py

```diff
@@ -33,6 +33,9 @@
 import pytest
 slow = pytest.mark.slow
 
+import logging
+#logging.getLogger().setLevel(logging.DEBUG)
```

Comment:
I kind of need to turn this on and off for testing every once in a while I don't always want to lookup the syntax for this. Do you have a better idea to leave it in without using a comment?